vpr/Makefile

```diff
@@ -39,7 +39,7 @@ OTHER_DIR := $(OTHER_DIR) \
 
 WARN_FLAGS = -Wall -Werror -Wpointer-arith -Wcast-qual -D__USE_FIXED_PROTOTYPES__ -ansi -pedantic -Wshadow -Wcast-align -D_POSIX_SOURCE -Wno-write-strings
 
-DEBUG_FLAGS = -g 
+DEBUG_FLAGS = -g -O0 -DDEBUG_INCREMENTAL_REROUTING
```

Comment:
Yup, we shouldn't leave that in the makefile as on by default for debug builds.  You could add a commended out line with additional DEBUG flags to show it exists.